zapadapter/worker/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"log"
+
+	"github.com/temporalio/samples-go/zapadapter"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/worker"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+func main() {
+	// The client and worker are heavyweight objects that should be created once per process.
+	c, err := client.NewClient(client.Options{
+		Logger: zapadapter.NewZapAdapter(NewZapLogger()),
+	})
```

Comment:
Probably worth calling attention to this with a more explicit comment since it's the key part